### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-tf-module.git`](https://github.com/cisagov/skeleton-tf-module.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/assessment-data-import-terraform.git assessment-data-import-terraform
    cd assessment-data-import-terraform
    git remote add skeleton https://github.com/cisagov/skeleton-tf-module.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the branch,
   `commit`, and `push` your changes:

    ```console
    git add README.md outputs.tf variables.tf 
    git commit --message="Resolve Lineage conflicts."
    git push --force --set-upstream origin lineage/skeleton
    ```

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍 👍

File: README.md [hunk 1]
```diff
@@ -2,92 +2,86 @@
...
-  on your system
-* [Terraform installed](
-  https://learn.hashicorp.com/terraform/getting-started/install.html)
-  on your system
+- [Terraform](https://www.terraform.io/) installed on your system.
+- AWS CLI access
+  [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
+  for the appropriate account on your system.
+- An accessible AWS S3 bucket to store Terraform state
+  (specified [here](backend.tf)).
```
Comment (jsf9k) [inline README.md]: Put these files names between backticks, please.

**Review by jsf9k — CHANGES_REQUESTED** [review]
I requested a few very minor changes.  Otherwise this looks great!

File: README.md (hunk 1 above)
Comment (mcdonnnj) [reply README.md]: Looking for clarification here. Do you mean change `[here](backend.tf)` to ``[`backend.tf`](backend.tf)``? It's used as a link in that line.

Comment (hillaryj) [reply README.md]: Yep, we're converting from the practice of linking "here" to actual text, i.e. `backend.tf` inside the brackets.

Comment (hillaryj) [reply README.md]: It's a Plain Language best-practice, iirc.

Comment (mcdonnnj) [reply README.md]: @jsf9k Please see 20ca527 for the changes in question. @hillaryj Thanks for confirming my suspicion. I'd assumed that was what he meant, but was looking for confirmation ^^

Comment (jsf9k) [reply README.md]: Heh, I actually misread the links the first time.  I like the changes, though.

I was wrong, but somehow that led to an improvement.  Praise Fortuna!

**Review by jsf9k — APPROVED** [review]

**Pull request merged**